README.rst

```diff
@@ -13,6 +13,16 @@ manifest (west.yml).
 Documentation
 *************
 
-Official latest documentation at https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/index.html
+Documentation is available at https://files.nordicsemi.com/artifactory/NCS/Wi-Fi/docs/
```

Comment:
Since this text is generic and not specific to any release, it should be kept as is.